src/app/(components)/button.tsx

```diff
@@ -49,6 +50,8 @@ function isButton(
   return !("href" in props);
 }
 
+export const DEFAULT_LOADING_MESSAGE = "Chargement, veuillez patientez...";
```

Comment:
```suggestion
export const DEFAULT_LOADING_MESSAGE = "Loading, please wait...";
```
Since i speak french i wrote this message in french, but it should be in english 😅